### PR TITLE
Expose public resize method without awt Dimension

### PR DESCRIPTION
**Pull request by Guy-Incognito**
In order to make resize work on GraalVM (which does not have java.awt.* classes) a resize method resize(int width, int height, boolean scale) is exposed.
Further static final variables have been renamed according to Java standards.

### CODE REVIEW

File: src/main/java/com/criteo/vips/Vips.java
Comment (warrenseine) [inline src/main/java/com/criteo/vips/Vips.java:83]: I've seen some code that would consider `os.name` to be "Darwin" on macOS. This doesn't seem to be true on my Macbook, but I'd prefer to avoid changing this.

File: src/main/java/com/criteo/vips/Vips.java
Comment (warrenseine) [inline src/main/java/com/criteo/vips/Vips.java:90]: Thanks, this is a nice check, the previous error message was a `NullReferenceException` IIRC. Can you make this change in a separate commit?

Comment (warrenseine) [inline src/main/java/com/criteo/vips/Vips.java:90]: Thanks, this is a nice check, the previous error message was a `NullReferenceException` IIRC. Can you make this change in a separate commit?

**Review by warrenseine — CHANGES_REQUESTED** [review]

File: src/main/java/com/criteo/vips/Vips.java
Comment (warrenseine) [reply src/main/java/com/criteo/vips/Vips.java]: Now I realize that the initial code was potentially bugged as well 😓 Anyway, let's not touch it!

Comment (Guy-Incognito) [reply src/main/java/com/criteo/vips/Vips.java]: Ok! I rolled back the change!

File: src/main/java/com/criteo/vips/Vips.java
Comment (Guy-Incognito) [reply src/main/java/com/criteo/vips/Vips.java:91]: Now in separate commit

Comment (Guy-Incognito) [reply src/main/java/com/criteo/vips/Vips.java:91]: Now in separate commit

**Review by warrenseine — APPROVED** [review]
LGTM, let me get a second review, and we'll merge it. Thanks!

File: src/main/java/com/criteo/vips/Vips.java
Comment (Guy-Incognito) [reply src/main/java/com/criteo/vips/Vips.java]: Haha, Thought so. They are semantically identical ;)

**Review by dbouron — APPROVED** [review]

**Pull request merged**
